### DISCUSSION TITLE
Open Selector in Custom Fields

### DISCUSSION THREAD

**Discussion by remchuk** [netbox-community/netbox#15431] [Q&A]
Hey there fellows!

I want to create a Custom field for an interface to be assigned to a different interface as part of a breakout implantation in our on-prem netbox.

The problem is that I can't seem to add an `Open Selector` to it.
When creating an IP address I'm able to assaing it to an interface on the right device by using the open selector:

![Screenshot from 2024-03-14 15-16-42](https://github.com/netbox-community/netbox/assets/133580699/516d9c95-bd2d-47c8-ac40-04deba1540b1)

Is there a way to implement in onto a custom field?

**Comment by remchuk** [comment]
Some perhaps?

**Reply by candlerb** [reply]
I think you need to explain your problem more clearly:
* Describe the environment: most importantly, what version of Netbox you are using.
* Show what you *did*. You said you created a custom field, so show how you defined it.
* Show what you *saw*.
* Show what you *expected to see* instead.

**Reply by remchuk** [reply]
I want to implement the `Open Selector Feature` onto custom fields.
I'm not sure what else can I describe.

All custom fields are without a `Open Selector`.
